### ISSUE TITLE
unknown directive "vod_upstream_location"

### ISSUE THREAD

**Issue by shaharmor** [kaltura/nginx-vod-module#205]
just installed following the readme and this is what i get:

```
service kaltura-nginx configtest
Testing nginx configuration: nginx: [emerg] unknown directive "vod_upstream_location" in /opt/kaltura/nginx/conf/nginx.conf:100
nginx: configuration file /opt/kaltura/nginx/conf/nginx.conf test failed
```

**Comment by shaharmor** [comment]
using version: 1.8.0-7

**Comment by shaharmor** [comment]
Looks like the README file points to 
`http://installrepo.kaltura.org/repo/apt/debian kajam main`

instead of the nightly, and the default one isn't updated

**Comment by shaharmor** [comment]
Actually, i'm not sure this is the reason cuz using nightly doesn't help either

**Comment by erankor** [comment]
The pre-compiled packages are still at an older version (before this change was added). 
We'll probably get to it in the next couple of weeks, but you can compile from master if you want it sooner

**Comment by shaharmor** [comment]
weeks? not sure we'll wait that long...
Can i build it with the latest nginx or i have to use the same version as you?

**Comment by erankor** [comment]
We are working with 1.8.0 (last time I checked it was the "stable" nginx version), didn't try it on newer versions. I expect it to work, but wasn't tested...

**Comment by csepulveda** [comment]
Until the last release (1.4) the directive is called vod_upstream instead vod_upstream_location. 
This last release was made in Nov.22 and vod_upstream_location was created in Dic 15: https://github.com/kaltura/nginx-vod-module/commit/c875dccba4a2e27e27423353eaaf502524a0ab1b

**Comment by jessp01** [comment]
FYI - a new deb package which includes all latest changes has been released to the 11.6.0 repo.
You can set:
\# cat /etc/apt/sources.list.d/kaltura.list 
deb [arch=amd64] http://installrepo.origin.kaltura.org/repo/apt/debian-nightly kajam main
to get it.
Don't forget to run:
\# aptitude update
afterwards.

# apt-cache policy kaltura-nginx 
kaltura-nginx:
  Installed: 1.8.0-8+b3
  Candidate: 1.8.0-8+b3
  Version table:
 *** 1.8.0-8+b3 0
        500 http://installrepo.origin.kaltura.org/repo/apt/debian-nightly/ kajam/main amd64 Packages
        100 /var/lib/dpkg/status

**Comment by erankor** [comment]
the new version was tagged and packaged, closing the issue
